src/prisma/schema.prisma

```diff
@@ -3304,7 +3304,7 @@ model WeeklyClaims {
   season         String
   week           String         @unique
   merkleTreeRoot String
-  totalClaimable Int
+  totalClaimable String
```

Comment:
there are 10 weekly claims in the DB, make sure to delete them if we are not using them before running the migration... (seems like we don't need them? why are there only 10? i dunno)

Reply:
weekly claims are created for each week of the season starting from preseason 02